### PR TITLE
feat: Allow field stone walls to be climable and support roofs

### PR DESCRIPTION
**Pull request by RobbieNeko**
### Required

- [x] I wrote the PR title in [conventional commit format](https://docs.cataclysmbn.org/en/contribute/changelog_guidelines/).
- [x] I ran the [code formatter](https://docs.cataclysmbn.org/en/contribute/contributing/#code-style).
- [x] I linked any relevant issues using [github keyword syntax](https://docs.cataclysmbn.org/en/contribute/contributing/#pull-request-notes) so it can be closed automatically.
- [x] I have [committed my changes to new branch that isn't `main`](https://docs.cataclysmbn.org/en/contribute/contributing/#make-your-changes) so it won't cause conflict when updating `main` branch later.

## Purpose of change

As I discovered in my own playthrough, for some reason field stone walls lacked the `SUPPORTS_ROOF` flag. This made them much less useful for building with, and is completely nonsensical given that dry stone walls absolutely support roofs irl.

## Describe the solution

Adds both the `SUPPORTS_ROOF` and the `CLIMBABLE` tag to field stone walls to bring them more in line with their mortared counterparts (in fact, I'd argue the field stone wall is *more deserving* of the latter than the mortared wall)

## Describe alternatives you've considered

- Cope

## Testing

It loads, it has the desired effects.

## Additional context

Building one's own house in game is already painful enough, we don't need arbitrary roofing restrictions

### CODE REVIEW

**Comment by RobbieNeko** [comment]
Wait, why the fuck did Autofix randomly decide it needs to fix a docs article? I didn't even touch the docs folder XD

**Comment by RoyalFox2140** [comment]
Yeahhhh now I have to fight a tree. Great.

File: doc/src/content/docs/en/i18n/reference/edge_cases.md [hunk 1]
```diff
@@ -24,4 +24,4 @@ Cataclysm: BN has more than 46,000 translatable strings (last updated at 2023-09
 be discouraged. The more translators there are, the easier it becomes!
 
 [^1]: Transifex API does not expose a public endpoint (at least without bearer token), so we were
-unable to update the number automatically.)
```
Comment (chaosvolt) [inline doc/src/content/docs/en/i18n/reference/edge_cases.md:27]: Goofy ahhh autofix

**Review by RoyalFox2140 — APPROVED** [review]
I'm going to let this get merged even if autofix is messing with documentation files. This can't possibly cause problems and if it does we can revert the changes to the docs.

File: doc/src/content/docs/en/i18n/reference/edge_cases.md (hunk 1 above)
Comment (RoyalFox2140) [reply doc/src/content/docs/en/i18n/reference/edge_cases.md:27]: Are you sure it's not going to just fix it again?

Comment (RobbieNeko) [reply doc/src/content/docs/en/i18n/reference/edge_cases.md:27]: I think he was just commenting haha

Comment (chaosvolt) [reply doc/src/content/docs/en/i18n/reference/edge_cases.md:27]: It was just a comment placed on the line change, yeah.

**Pull request merged**